public/index.html

```diff
@@ -196,12 +196,13 @@
   <body>
     <noscript> You need to enable JavaScript to run this app. </noscript>
     <div id="root"></div>
-  </body>
-  <script>
-    var iOS = !!navigator.platform && /iPhone/.test(navigator.platform)
+    <script>
```

Comment:
fixes `Failed to construct WebSocket: invalid url`

this `script` tag should be **inside** `body because react adds bundle scripts **before** the closing body tag.
this is important for local debug because the source of the **last script** is used by dev server.

if to put some custom script after body it will the last script in the document so dev server client will get undefined reading its src which will cause an exception